generators/http-loader.js

```diff
@@ -3,32 +3,37 @@ const chalk = require('chalk');
 const axios = require('axios');
 const generate = require('./generator');
 const SchemaLoader = require('./schema-loader');
+const config = require('../config');
 
 function postRecord (host, port, path, data) {
-    return axios.post(`${host}:${port}/${path}`, data);
+    axios.defaults.headers.common['Authorization'] = 'Bearer ' + config.token
+    return axios.put(`${host}:${port}/${path}`, data);
```

Comment:
Could we have this be an option that is passed?  Instead of it always being a put?